### ISSUE TITLE
`get_test_cases` reorders list of test cases

### ISSUE THREAD

**Issue by Thalley** [auto-pts/auto-pts#1320]
If the autoptsclient is called with `-c TEST_B TEST_A` then after `get_test_cases` `self.args.test_cases` becomes `["TEST_A", "TEST_B"]`. 

If a developer is debugging a test case and needs to run some tests in a specific order, they should be able to control the order when using the autoptsclient.

**Comment by mringwal** [comment]
Good point. However, the order of tests should not matter, otherwise, something else is wrong :)

**Comment by Thalley** [comment]
> Good point. However, the order of tests should not matter, otherwise, something else is wrong :)

For sure - And something is in fact wrong, and I needed to reorder the test to prove that the order of tests mattered :D

**Comment by Thalley** [comment]
See also https://discord.com/channels/720317445772017664/1316767993522421830/1316800022981120114
